src/mpcCoreKit.ts

```diff
@@ -1012,7 +1012,17 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     return tssNonce;
   }
 
-  private async setupTkey(providedImportTssKey?: string, isSfaKey?: boolean): Promise<void> {
+  private async setupTkey(
+    providedImportTssKey?: string,
+    sfaLoginResponse?: TorusKey | TorusLoginResponse | TorusAggregateLoginResponse,
```

Comment:
Do we need the sfaLoginResponse ?
can we just pass in the sfaMetadataUpgraded boolean?

Reply:
yeah i thought about it but it will just increase the number of random variables in the function, its better to pass loginResponse directly in case we need to access any other properties of login response in future